chains/base/metadata.yaml

```diff
@@ -4,6 +4,14 @@ blockExplorers:
     family: etherscan
     name: BaseScan
     url: https://basescan.org
+  - apiUrl: https://api.routescan.io/v2/network/mainnet/evm/8453/etherscan/api
+    family: routescan
+    name: Base Explorer
+    url: https://base.thesuperscan.io/
+  - apiUrl: https://base.blockscout.com/api
+    family: blockscout
+    name: Base Mainnet explorer
+    url: https://base.blockscout.com/
```

Comment:
can you omit trailing `/` from urls where possible pls

Reply:
done!